extensions/interactions/DragAndDropSort/directives/drag_and_drop_sort_interaction_directive.html

```diff
@@ -0,0 +1,63 @@
+<style>
+  .drag-and-drop-sort-form-container {
+    background: #fff;
+    max-width: 100%;
+    padding: 8px;
+  }
+  @media screen and (max-width: 959px) {
+    .drag-and-drop-sort-form-container {
+      width: 100vw;
+    }
+  }
+  .angular-ui-tree-placeholder {
+    width: 200px;
+    height: 70px;
+    border: 2px dashed lightblue;
+    margin-left: 50px;
+  }
+  .drag-and-drop-sort-container {
+    border: 1px solid grey;
+    overflow: scroll;
+    margin: auto;
+  }
+  .item {
+    border: 1px solid grey;
+    width: 200px;
+    height: 70px;
+    background-color: #add8e6;
+    margin: 10px 0 0 70px;
+  }
+  .subItem {
```

Comment:
`.subitem` since it breaks the case conventions.

Reply:
Done